tests/test_edgeql_ir_scopetree.py

```diff
@@ -37,8 +37,13 @@ class TestEdgeQLIRScopeTree(tb.BaseEdgeQLCompilerTest):
                           'cards.esdl')
 
     UUID_RE = re.compile(
```

Comment:
Why is it called UUID_RE if it really matches something else?

Reply:
It used to match uuid literals exactly, but it now matches something else. I'll rename.